src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -428,6 +429,9 @@ public function load(array $configs, ContainerBuilder $container)
         if (!$config['disallow_search_engine_index'] ?? false) {
             $container->removeDefinition('disallow_search_engine_index_response_listener');
         }
+
+        $container->registerForAutoconfiguration(ServiceRouterLoaderInterface::class)
+            ->addTag('routing.router_loader');
```

Comment:
but adding the tag doesn't work, because the tag needs an attribute right now, does it?
that's why removing the attribute on the tag might be better also?

Reply:
The tag works because the attribute `service_id` is a fallback. The default behavior is to use the FQCN.